torch/testing/_internal/distributed/nn/api/remote_module_test.py

```diff
@@ -505,6 +504,16 @@ def test_send_remote_module_over_the_wire(self):
                 dst_worker2_name, remote_forward_async, (remote_module, args)
             )
             self.assertEqual(ret2, tuple(reversed(args)))
+            
+            args = True
+            ret3 = rpc.rpc_sync(dst_worker2_name, remote_train, (remote_module, args))
+            self.assertEqual(ret3.training, True)
+            
+            ret4 = rpc.rpc_sync(dst_worker2_name, remote_eval, (remote_module,))
+            self.assertEqual(ret4.training, False)
```

Comment:
Probably add a separate test method for this instead of using this. Also, what you probably want to test is something like this:

1. Run `remote_module.train()` and `remote_module.eval()` locally
2. Use RPC to retrieve the training parameter remotely, you can define a function like this and call it over RPC:

```
def get_training_arg(module_rref):
  return module_rref.local_value().training

# Call it
rpc.rpc_sync(dst, get_training_arg, args=(remote_module.get_module_rref()))